src/ops/fuelup_toolchain/export.rs

```diff
@@ -0,0 +1,322 @@
+    const INVALID_CHANNEL: &str = "my-channel";
+
+    fn remove_toolchain_info() {
+        if let Some(fuel_toolchain_toml_file) = get_fuel_toolchain_toml() {
+            if fuel_toolchain_toml_file.exists() {
+                fs::remove_file(fuel_toolchain_toml_file).unwrap();
+            }
+        }
+    }
+    fn create_toolchain_info() {
+        if let Some(fuel_toolchain_toml_file) = get_fuel_toolchain_toml() {
+            if !fuel_toolchain_toml_file.exists() {
+                fs::create_dir_all(fuel_toolchain_toml_file.parent().unwrap()).unwrap();
+                fs::File::create(&fuel_toolchain_toml_file).unwrap();
+            }
+        } else {
+            fs::File::create(FUEL_TOOLCHAIN_TOML_FILE).unwrap();
+        }
+    }
+    // mock setting.toml
+    fn create_toolchain_settings_file() {
+        let setting_file_path = settings_file();
+        if !setting_file_path.exists() {
+            fs::create_dir_all(setting_file_path.parent().unwrap()).unwrap();
+            let mut file = fs::File::create(&setting_file_path).unwrap();
+            file.write_all(b"default_toolchain = \"latest\"").unwrap();
+        }
+    }
+    fn assert_channel_name(expected: &String) {
+        let toolchain_override =
+            ToolchainOverride::from_project_root().expect("toolchain override");
+        let _actual = toolchain_override.cfg.toolchain.channel.to_string();
+        assert!(matches!(expected, _actual));
+    }
+
+    #[test]
+    #[serial]
+    fn export_toolchain_with_exists_toolchain_info_with_cancel() {
+        create_toolchain_settings_file();
+        create_toolchain_info();
+        export(
+            ExportCommand {
+                name: Some(DistToolchainName::Beta3.to_string()),
+                channel: None,
+                force: false,
+            },
+            &INPUT_NO[..],
+        )
+        .unwrap();
+    }
+
+    #[test]
+    #[serial]
+    #[should_panic]
+    fn export_toolchain_with_invalid_channel_provided_throws_err() {
+        create_toolchain_settings_file();
+        remove_toolchain_info();
+        export(
+            ExportCommand {
+                name: Some(DistToolchainName::Beta3.to_string()),
+                channel: Some(INVALID_CHANNEL.to_string()),
+                force: false,
+            },
+            &INPUT_INVALID_CHANNEL[..],
+        )
+        .unwrap();
+    }
+
+    #[test]
+    #[serial]
+    #[should_panic]
+    fn export_toolchain_with_invalid_channel_inputted_throws_err() {
+        create_toolchain_settings_file();
+        remove_toolchain_info();
+        export(
+            ExportCommand {
+                name: Some(DistToolchainName::Latest.to_string()),
+                channel: None,
+                force: false,
+            },
+            &INPUT_INVALID_CHANNEL[..],
+        )
+        .unwrap();
+    }
+
+    #[test]
+    #[serial]
+    fn export_toolchain_with_exists_toolchain_info() {
```

Comment:
```suggestion
    fn export_toolchain_with_forced_overwrite() {
```

Reply:
Looking at this and then the following test below I would split them into unit tests because there's more than 1 test case in each.